src/v/test_utils/randoms.h

```diff
@@ -39,4 +46,104 @@ random_broker(int32_t id_low_bound, int32_t id_upper_bound) {
         .cores = random_generators::get_int<uint32_t>(96)});
 }
 
-} // namespace tests
\ No newline at end of file
+inline bool random_bool() {
+    static thread_local std::random_device
+      rd; // Will be used to obtain a seed for the random number engine
+    static thread_local std::mt19937 gen(
+      rd()); // Standard mersenne_twister_engine seeded with rd()
+    std::uniform_int_distribution<> dist(0, 100);
+    return dist(gen) > 50;
```

Comment:
why are we creating a new thread local random source for this? other methods in here use the existing random generators utilities.